pkg/operator/operator.go

```diff
@@ -165,10 +165,12 @@ func (o *Operator) Start(ctx context.Context) {
 		defer wg.Done()
 		lo.Must0(o.Manager.Start(ctx))
 	}()
-	wg.Add(1)
-	go func() {
-		defer wg.Done()
-		sharedmain.MainWithConfig(ctx, "webhook", o.GetConfig(), o.webhooks...)
-	}()
+	if injection.GetOptions(ctx).WebhookPort <= 0 {
```

Comment:
I think I like `DISABLE_WEBHOOK` better than this. This is kind of equivalent to a magic number and I don't think I've used something that is configured in this way. Since we default the port number, this isn't equivalent to a non-configured parameter. 

The condition is also inverted right?